css-text/Overview.bs

```diff
@@ -1296,7 +1299,7 @@ Line Breaking Details</h3>
 
 <pre class='propdef'>
 Name: overflow-wrap, word-wrap
-Value: normal | break-word
+Value: normal | break-word | break-spaces
```

Comment:
Right. I've pinged the WG now, let's see what comes out of this.
https://lists.w3.org/Archives/Public/www-style/2016May/0134.html